src/sentry/tasks/post_process.py

```diff
@@ -360,6 +360,7 @@ def post_process_group(
         # This should eventually be completely removed and transactions
         # will not go through any post processing.
         if is_transaction_event:
+            record_transaction_name_for_clustering(event.project, event)
```

Comment:
@lynnagara This is where we hook into post-processing to write transaction names to redis, as discussed in yesterday's meeting.